test-fixtures/perfTestTransform.py

```diff
@@ -0,0 +1,22 @@
+import json
+
+PERFHERDER_DATA = {"framework": 'mozperftest',
+                   "application": {"name": 'firefox-ios'},
+                   "suites": []
+                   }
+
+with open('test.json') as json_file:
+    data = json.load(json_file)
+    for p in data:
+        suite = {}
+        suite["name"] = p["testName"]
+        suite["subtests"] = []
+        for key, value in p.items():
+            if key != "testName":
+                subtest = {}
+                subtest["name"] = key
+                subtest["replicates"] = [value]
+                suite["subtests"].append(subtest)
+        PERFHERDER_DATA["suites"].append(suite)
+
+print("PERFHERDER_DATA:", json.dumps(PERFHERDER_DATA))
```

Comment:
We may need to expose the PERFHERDER_DATA... it is shown in the Bitrise logs for the `Create perfherder data object` steep, but not in any artifact generated so I'm not totally sure it will be directly availble to Taskcluster. Something to check / ask

Reply:
Let's keep this in mind for a possible follow up, but we can move forward for now

Reply:
Once we have this running on Taskcluster we will have more information to preceed. I would say we are good now